File: 1180/1180.cpp
#include <iostream>
#include <vector>

int main(){
    int n, min, pos;
    std::cin >> n;

    int vet[n];

    for (int i=0;i<n;i++){
        std::cin >> vet[i];
    }

    min = vet[0];
    for (int i=0;i<n;i++){
        if (vet[i] < min) {
            min = vet[i];
            pos = i;
        }
    }

    std::cout << "Menor valor: " << min << "\n";
    std::cout << "Posicao: " << pos << "\n";    

    return 0;
}

File: 1180/1180.rs
use std::io;

fn input() -> String {
    let mut entrada : String = String::new();
    io::stdin()
        .read_line(&mut entrada)
        .expect("Erro ao ler a entrada!");
    entrada.to_string()
}

fn main() {
    let entrada : String = input();
    let n : usize = entrada.trim().parse().expect("msg");

    // valores mutaveis
    let mut min : i32;
    let mut pos : usize = 0; 

    let valores : String = input();
    let valores_arr : Vec<i32> = valores
        .split_whitespace()
        .take(n)
        .map(|s| s.parse::<i32>().expect("msg"))
        .collect();

    min = valores_arr[0];
    for i in 0..n{
        if valores_arr[i] < min{
            min = valores_arr[i];
            pos = i;
        }
    }

    println!("Menor valor: {}", min);
    println!("Posicao: {}", pos);
}